### PR TITLE
Create README for Each Package

### PR DESCRIPTION
**Pull request by brhook-aws**
#### Major
* Create and update README to each package in the repository, including the base
* Add in LICENSE to each package

#### Minor
* Clean up webpack.common.js/webpack.test.js files to only export one item, and the karma.conf.js to only then expect one item coming in
* Fix pathing mistake in jsdoc.conf.json
* Remove "docs" command from the sub-packages, keeping only in root
* Add in jsconfig.json so VS Code can understand dependencies easier

### Why
With the splitting of the packages, we will need a unique README and guidance, rather than a generalized one in the root. This is especially so considering each package will be published to npm standalone, and won't be part of the repo when viewed there. Likewise, each package will need its own LICENSE to standalone.  

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,1981 +1,97 @@
...
+The easiest way to start using the API is to include the build file that corresponds with the rendering engine you are using. See the [Getting Started](#Getting-Started) guide for a walkthrough using this method and the [API Documentation](https://aws-samples.github.io/amazon-sumerian-hosts/) for more detailed information on the classes and methods available. Amazon Sumerian Hosts has all three of its packages published to [npm](https://www.npmjs.com/), so alternatively you can install in an existing Node.js project by running `npm install --save-dev @amazon-sumerian-hosts/core @amazon-sumerian-hosts/three @amazon-sumerian-hosts/babylon`.
```
Comment (kris-work) [inline README.md]: The "Getting Started" link is broken and the wording doesn't quite make sense now that the "Getting Started" guide has been split into separate Babylon and Three "Gettting Started" guides. In this main readme, I would call out each Getting Started guide individually and link to both of them.

File: packages/amazon-sumerian-hosts-babylon/README.md [hunk 2]
```diff
@@ -0,0 +1,970 @@
...
+
+## License
+
+This library is licensed under the MIT-0 License. See the [LICENSE](LICENSE) file. The assets within examples are licensed under the CC-BY-4.0 License. See the [LICENSE](examples/assets/LICENSE) file.
+<br/><br/>
+
+# [Getting Started](#Getting-Started)
+
+This guide steps through the creation of [examples/babylon.html](examples/babylon.html), resulting in an interactive 3D character that animates and speaks the text you input into the textarea, rendered in the Web rendering engine of your choice.
```
Comment (kris-work) [inline packages/amazon-sumerian-hosts-babylon/README.md:16]: "steps through the creation" is a little confusing to me. It makes it sound like we're asking the user to write the code themselves. Also, some of the other wording (e.g. "rendering engine of your choice") isn't appropriate. I suggest changing this to 

> This guide steps you through the code of the `examples/babylon.html` file, a working example demonstrating the features of the Sumerian Host characters running in Babylon.js

File: packages/amazon-sumerian-hosts-three/README.md [hunk 3]
```diff
@@ -0,0 +1,1000 @@
...
+
+## License
+
+This library is licensed under the MIT-0 License. See the [LICENSE](LICENSE) file. The assets within examples are licensed under the CC-BY-4.0 License. See the [LICENSE](examples/assets/LICENSE) file.
+<br/><br/>
+
+# [Getting Started](#Getting-Started)
+
+This guide steps through the creation of [examples/three.html](examples/three.html), resulting in an interactive 3D character that animates and speaks the text you input into the textarea, rendered in the Web rendering engine of your choice.
```
Comment (kris-work) [inline packages/amazon-sumerian-hosts-three/README.md:16]: I have the same request as I did with the Bablyon README. "steps through the creation" is a little confusing to me. It makes it sound like we're asking the user to write the code themselves. Also, some of the other wording (e.g. "rendering engine of your choice") isn't appropriate. I suggest changing this to 

> This guide steps you through the code of the `examples/three.html` file, a working example demonstrating the features of the Sumerian Host characters running in three.js

**Review by kris-work — CHANGES_REQUESTED** [review]

File: packages/amazon-sumerian-hosts-babylon/README.md (hunk 2 above)
Comment (brhook-aws) [reply packages/amazon-sumerian-hosts-babylon/README.md:16]: I think the initial expectation is that users would write the code from `Getting Started in {engine}`, but use the example as a reference for missing pieces of the makeup of that file. That said, I'll rephrase here, as I agree it is awkward.

**Review by kris-work — APPROVED** [review]

**Review by haozhao-aws — APPROVED** [review]

**Pull request merged**